x-pack/plugins/enterprise_search/public/applications/applications/components/engine/engine_schema.tsx

```diff
@@ -346,6 +346,31 @@ export const EngineSchema: React.FC = () => {
   return (
     <>
       <EuiFlexGroup direction="column" gutterSize="l">
+        {hasSchemaConflicts && (
+          <EuiCallOut
+            title={i18n.translate(
+              'xpack.enterpriseSearch.content.applications.schema.conflictsCallOut.title',
+              { defaultMessage: 'Potential field mapping issues found' }
+            )}
+            iconType="error"
+            color="danger"
+          >
+            <p>
+              <FormattedMessage
+                id="xpack.enterpriseSearch.content.applications.schema.conflictsCallOut.description"
+                defaultMessage="Schema field type conflicts can be resolved by navigating to the source index directly and updating the field type of the conflicting field(s) to match that of the other source indices."
+              />
+            </p>
+            {!onlyShowConflicts && (
+              <EuiButton color="danger" fill onClick={toggleOnlyShowConflicts}>
+                <FormattedMessage
+                  id="xpack.enterpriseSearch.content.applications.schema.conflictsCallOut.button"
+                  defaultMessage="View conflicts"
+                />
+              </EuiButton>
+            )}
+          </EuiCallOut>
+        )}
         <EuiFlexGroup>
```

Comment:
` <EuiFlexGroup alignItems="center" justifyContent="spaceBetween">` to vertically center switch and filter and then push them to the far ends

Reply:
you're sneaky! not really part of this work but for you, i'm happy to oblige haha

Reply:
I know! I wasn't going to do it but it's like a splinter in my miiiiinnndddd 🙇‍♂️

Plus the show conflicts toggle switch is in there so, you know, it's not _completely_ unrelated 🙏

Reply:
✅ https://github.com/elastic/kibana/pull/157282/commits/e1f730dadd9f5b02c9f856b6b168f047551cbc50